src/ui/components/transactions/SetScript/SetScriptCardHeader.module.css

```diff
@@ -0,0 +1,21 @@
+.header {
+  align-items: center;
+  display: flex;
+}
+
+.icon {
+  height: 48px;
+  margin-right: 10px;
+  width: 48px;
+}
+
+.data {
+  color: #9ba6b2;
```

Comment:
You can use `var(--color-basic500)` here instead. Colors are declared in `app.styl`